scalableminds/webknossos
# Do not poll jobs from dashboard if jobs are disabled

### URL of deployed dev instance (used for testing):
- Jobs status is no longer polled if jobs are not enabled, avoiding backend logging spam
- If jobs requests are sent despite jobs being enabled, the backend routes will return bad request

### Steps to test:
- open dashboard, see backend logging and/or browser network tab
- no polling should happen
- after enabling jobs, the polling should happen again

### Issues:
- fixes #5752 

------
